config/flags.go

```diff
@@ -1,32 +1,31 @@
-package flags
+package config
 
 import (
-	"github.com/Layr-Labs/eigenda-proxy/flags/eigendaflags"
+	"github.com/Layr-Labs/eigenda-proxy/config/eigendaflags"
+	eigenda_v2_flags "github.com/Layr-Labs/eigenda-proxy/config/eigendaflags/v2"
+	"github.com/Layr-Labs/eigenda-proxy/store"
+
 	"github.com/Layr-Labs/eigenda-proxy/logging"
 	"github.com/Layr-Labs/eigenda-proxy/metrics"
-	"github.com/Layr-Labs/eigenda-proxy/store"
 	"github.com/Layr-Labs/eigenda-proxy/store/generated_key/memstore"
 	"github.com/Layr-Labs/eigenda-proxy/store/precomputed_key/redis"
 	"github.com/Layr-Labs/eigenda-proxy/store/precomputed_key/s3"
-	"github.com/Layr-Labs/eigenda-proxy/verify"
-
+	"github.com/Layr-Labs/eigenda-proxy/verify/v1"
 	"github.com/urfave/cli/v2"
 
 	"github.com/Layr-Labs/eigenda-proxy/common"
 )
 
 const (
-	EigenDAClientCategory      = "EigenDA Client"
-	LoggingFlagsCategory       = "Logging"
-	MetricsFlagCategory        = "Metrics"
-	EigenDADeprecatedCategory  = "DEPRECATED EIGENDA CLIENT FLAGS -- THESE WILL BE REMOVED IN V2.0.0"
-	MemstoreFlagsCategory      = "Memstore (for testing purposes - replaces EigenDA backend)"
-	StorageFlagsCategory       = "Storage"
-	StorageDeprecatedCategory  = "DEPRECATED STORAGE FLAGS -- THESE WILL BE REMOVED IN V2.0.0"
-	RedisCategory              = "Redis Cache/Fallback"
-	S3Category                 = "S3 Cache/Fallback"
-	VerifierCategory           = "KZG and Cert Verifier"
-	VerifierDeprecatedCategory = "DEPRECATED VERIFIER FLAGS -- THESE WILL BE REMOVED IN V2.0.0"
+	EigenDAClientCategory   = "EigenDA Client"
+	EigenDAV2ClientCategory = "EigenDA V2 Client"
+	LoggingFlagsCategory    = "Logging"
+	MetricsFlagCategory     = "Metrics"
+	MemstoreFlagsCategory   = "Memstore (for testing purposes - replaces EigenDA backend)"
```

Comment:
add a note that this category contains flags for both v1 and v2 memstore (I thin it does?)

Reply:
I believe that most of these categories contain flags for both v1 and v2, so I'm not sure if that would be a helpful comment here

Reply:
Sorry this is github review UI being weird. Was only highlighting line 24. Because memstore is a backend for which we have 2 versions. we have eigendav1 and eigendav2 categories, so why not for memstore?

Reply:
> Was only highlighting line 24

Yep, understood. My point is just that most of these other categories also have v1 and v2 things mixed together, so it's not a unique issue with the memstore category. This is an argument for cleaning up the flag config system, for sure

> so why not for memstore?

I'm not sure the reason that memstore is combined to a single category.